src/uikit.cs

```diff
@@ -16085,4 +16153,256 @@ interface UITextAttributesConstants {
 		[Field ("UITextAttributeTextShadowOffset")]
 		NSString TextShadowOffset { get; }
 	}
+
+#region Drag and Drop
+	interface IUIInteraction {}
+	interface IUIDropSession {}
+	interface IUIDragDropSession {}
+	interface IUIDragAnimating {}
+	interface IUIDragSession {}
+	interface IUIDropInteractionDelegate {}
+		
+	[NoWatch, NoTV, iOS (11,0)]
+	[Protocol]
+	interface UIDragAnimating
+	{
+		[Abstract]
+		[Export ("addAnimations:")]
+		void AddAnimations (Action animations);
+	
+		[Abstract]
+		[Export ("addCompletion:")]
+		void AddCompletion (Action<UIViewAnimatingPosition> completion);
+	}
+	
+	[NoWatch, NoTV, iOS (11,0)]
+	[Protocol]
+	interface UIDragDropSession
+	{
+		[Abstract]
+		[Export ("items")]
+		UIDragItem[] Items { get; }
+	
+		[Abstract]
+		[Export ("locationInView:")]
+		CGPoint LocationInView ([NullAllowed] UIView view);
+	
+		[Abstract]
+		[Export ("allowsMoveOperation")]
+		bool AllowsMoveOperation { get; }
+	
+		[Abstract]
+		[Export ("restrictedToDraggingApplication")]
+		bool RestrictedToDraggingApplication { [Bind ("isRestrictedToDraggingApplication")] get; }
+	
+		[Abstract]
+		[Export ("hasItemsConformingToTypeIdentifiers:")]
+		bool HasItemsConformingToTypeIdentifiers (string[] typeIdentifiers);
+	
+		[Abstract]
+		[Export ("canLoadObjectsOfClass:")]
+		bool CanLoadObjectsOfClass (INSItemProviderReading aClass);
```

Comment:
Let's check what Swift is using, they normally use API name very close to the .net we have been promoting for years (and it also makes ease googl'ability)